go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.27.6
-	github.com/osrg/gobgp/v3 v3.14.0
+	github.com/osrg/gobgp/v3 v3.14.1-0.20230525155627-5719c6d2edba
```

Comment:
Shall we wait a couple of days for 3.15?

Reply:
Yeah, in this case, maybe we can wait with merging this PR until the next GoBGP release (which should come in about a week time-frame), as it conflicts with https://github.com/cilium/cilium/pull/25660 anyways.

But during the last sig-traffic-eng meeting we discussed that there might be nothing bad on staying on an unreleased version of GoBGP for a while, as it at least allows us to test recent changes in the meantime. We would bump to the next minor version as soon it is released in that case.

Reply:
Seems like they released v3.14.0. Shall we change?

Reply:
@YutaroHayakawa 
> Seems like they released v3.14.0. Shall we change?

`v3.14.0` is one month old, we are waiting for `v3.15.0` :) Already asked in Slack about when the next release will be.

Reply:
Oops 🤦

Reply:
GoBGP `v3.15.0` is out now, I just updated the PR.